filebeat/input/filestream/filestream.go

```diff
@@ -53,6 +54,8 @@ type logFile struct {
 	closeRemoved  bool
 	closeRenamed  bool
 
+	isInactive bool
```

Comment:
Since this is updated by two goroutines you should make it an atomic to be safe.

Reply:
You've convinced me that you don't need explicit synchronization but you want the CPU helping to ensure the reads+writes happen in the way you think they should.